Makefile

```diff
@@ -64,7 +67,10 @@ autoformat: .venv/bin/python .venv/bin/black .venv/bin/elastic-ingest
 	.venv/bin/ruff scripts --fix
 
 test: .venv/bin/pytest .venv/bin/elastic-ingest
-	.venv/bin/pytest --cov-report term-missing --cov-fail-under 92 --cov-report html --cov=connectors --fail-slow=$(SLOW_TEST_THRESHOLD) -sv tests
+	.venv/bin/pytest --cov-report term-missing --cov-fail-under 92 --cov-report html --cov=connectors --fail-slow=$(SLOW_TEST_THRESHOLD) -sv tests --ignore tests/agent
+
+test-agent: .venv/bin/pytest .venv/bin/elastic-ingest install-agent
```

Comment:
I think we should add this to CI now, to avoid forgetting or letting it get too behind in coverage. I _think_ that ElasticMachine should be able to clone the private repo. Hit me up if it's not working.